README.md

```diff
@@ -30,7 +30,7 @@ After you're finished please remove all the comments and instructions!
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
Great job adding the Microverse README Template. But let's customize it. 👍   Kindly modify your README file to match your project and remove sections that don't apply.
- Please remove all the instructions targeting you. For example: 
>"Describe how a new developer could make use of your project." 
  or
> NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.
-  Please remove the template logo on the top, if you have your own logo you can replace it with this one, otherwise please delete it. This is crucial to have a professional and instructive repository. 👌 
- Please make sure your license link works fine. Currently, it gives 404. 👍

Reply:
@huseyinbiyik Is it necessary to rewrite all the commits or can I  leave them how they are,  just for this time

Reply:
hey @danielamoreno699, sorry for the late answer. Yes, it was for the next commits. And your commit messages are brilliant right now 👍